### PR TITLE
Display recording options for current recording list

### LINKED ISSUE #127: [design] Recording's options (max age, max size, toDisk) should be discoverable
The API responses from `/api/v1/targets/:targetId/recordings` includes details on each recording about their maxAge/maxSize/toDisk option settings, but there is no UI element that exposes this information to the user. Since #103 adds a way for users to customize these settings and change their values from the defaults when a recording is created, there should also be a corresponding way for the user to see the values of these settings for each recording.

### PR DESCRIPTION
**Pull request by Alexjsenn**
Fixes #127

### CODE REVIEW

**Comment by Alexjsenn** [comment]
@andrewazores Do you think it makes sense to change the recording report's appearance? Or is it unecessary?

**Comment by andrewazores** [comment]
Change the appearance in what sense or for what purpose?

**Comment by Alexjsenn** [comment]
Making the titles (at least) a bit smaller, so that they aren't so much larger than the rest of the text.

**Comment by andrewazores** [comment]
I don't think it's necessary at this point, but if you don't find any other good work items to take up then that could be something to look at. That report transformation is done on the ContainerJFR backend, however, so it wouldn't be part of this PR.

**Comment by Alexjsenn** [comment]
Ok, makes sense!

File: src/app/RecordingList/ActiveRecordingsList.tsx [hunk 1]
```diff
@@ -219,6 +219,16 @@ export const ActiveRecordingsList: React.FunctionComponent<ActiveRecordingsListP
...
         >
+          <Text>Recording Options:</Text>
+          <Text>
+            toDisk = { String(props.recording.toDisk) } &emsp; 
+            maxAge = {props.recording.maxAge / 1000}s &emsp; 
+            maxSize = { props.recording.maxSize }B
+          </Text>
+          <br></br>
+          <hr></hr>
+          <br></br>
```
Comment (andrewazores) [inline src/app/RecordingList/ActiveRecordingsList.tsx:230]: `hr` and `br` are empty elements in HTML, so these should simply be `<br>` and `<hr>` with no `</br>` closing tags.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br

Comment (Alexjsenn) [reply src/app/RecordingList/ActiveRecordingsList.tsx:230]: Im trying to understand how the Text component works, and it seemes like it should work but for some reason it gives an error when I don't close hr and br.

Comment (andrewazores) [reply src/app/RecordingList/ActiveRecordingsList.tsx:230]: Hmm. Maybe it's a quirk of TSX syntax, or the way that it's compiled. No problem then.

**Review by andrewazores — APPROVED** [review]

**Pull request merged**